### PR TITLE
[EAG-321] Fix mergeObjects.

### PR DESCRIPTION
**Pull request by rcarranza-carecloud**
**Ticket:** https://jira.carecloud.com/browse/EAG-321
**Changes:**
 - Fix `mergeObjects` method by using lodash `mergeWith` instead.

**Reviewers:** @jnorris-carecloud , @jleyba-carecloud

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -198,21 +199,14 @@ export function isObject(thing) {
   return typeof thing === 'object' && thing !== null && !Array.isArray(thing);
 }
 
+function mergeArrays(objValue, srcValue) {
```
Comment (amartin-carecloud) [inline src/utils.js]: Should this function only return something if an array exists?

Comment (rcarranza-carecloud) [reply src/utils.js]: If `objValue` is an array the function will concat `srcValue` to `objValue`. If `objValue` is not an array `srcValue` will override `objValue`.
I'll make a fix to perform the concat if and only if both values are arrays.

Comment (jnorris-carecloud) [reply src/utils.js]: We're discussing a offline about this and trying a different approach.

Comment (rcarranza-carecloud) [reply src/utils.js]: We decided to add a simple check at the beginning of the function to check the value of `obj1`

**Review by jnorris-carecloud — APPROVED** [review]

File: src/utils.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import 'setimmediate';
 import validateFormData from './validate';
+import { isObject as _isObject } from 'lodash';
```
Comment (amartin-carecloud) [inline src/utils.js:4]: why the alias?

Comment (jnorris-carecloud) [reply src/utils.js:4]: `isObject` is declared in this file, it's an already existing helper from RJSF.

**Pull request merged**